### DISCUSSION TITLE
Site Import - Multi Section Field Errors

### DISCUSSION THREAD

**Discussion by adfrost81** [netbox-community/netbox#9680] [Q&A] [answered]
I'm having a hell of a time getting the Site data to import. The Multi-Selection fields are giving me fits. I keep getting errors like this:

_Row 1 cf_st_dept_printers: Select a valid choice. HP LaserJet M605N is not one of the available choices._

This is how my data is formatted for that field in the CSV: `"HP LaserJet M602N, HP LaserJet M605N"`

This is the list of accepted values for the field, and as you can see, both values match perfectly:

![2022-07-07 10_37_41-Window](https://user-images.githubusercontent.com/91818709/177814365-adbe8789-47be-4a15-9f1a-eea7b34c7eb9.png)

I tried changing it to only one value ( HP LaserJet M605N) and it successfully imported, but I need to be able to import these with multiple values.

The instance of NetBox is 66648125966c (v3.2.3)

**Comment by jeremystretch** [comment, marked as answer]
It might be the space trailing the comma.

**Reply by adfrost81** [reply]
I just realized I had tried it without spaces on our older instance (3.1.0), but not on the newer (3.2.3). It does import successfully without the spaces on the newer version. Thanks for the help!
